client/src/App.tsx

```diff
@@ -196,7 +196,11 @@ function App() {
 			case 'weather':
 			case 'roomDirectory':
 			case 'settings':
+<<<<<<< HEAD
```

Comment:
this is due to a merge conflict, need to fix this line since it's not actual code